src/shared/format-table.c

```diff
@@ -262,6 +262,9 @@ static size_t table_data_size(TableDataType type, const void *data) {
         case TABLE_BOOLEAN:
                 return sizeof(bool);
 
+        case TABLE_BOOLEAN_CHECKMARK:
```

Comment:
merge with the preceding block, i.e. see how TABLE_TIMESTAMP/TABLE_TIMESTAMP_UTC/… do it